### PR TITLE
Markdownlint problem matcher

### PR DESCRIPTION
**Pull request by nschonni**
## Summary
Allows the lines to be directly flagged in the GitHub UI when there is a markdownlint issue

### CODE REVIEW

**Comment by nschonni** [comment]
I added a commit to show how it looks when files aren't touched in the PR, but have flagged issues. I'll rebase it off and undraft if this looks OK

**Comment by nschonni** [comment]
In case you can't see it because it's a beta feature
image

**Review by Thraka — APPROVED** [review]
I like it.

**Review by tdykstra — CHANGES_REQUESTED** [review]
This PR enables MD004 but we discussed this as a team and decided to leave it turned off.

File: .markdownlint.json [hunk 1]
```diff
@@ -1,7 +1,6 @@
 {
     "default": true,
     "MD001": false,
-    "MD004": false,
```
[the remark this answers is not in the document]
Comment (nschonni) [reply .markdownlint.json:4]: Yes, this was just added in a WIP commit to show the matching

**Comment by nschonni** [comment]
Rebased out the test commit and marked this ready for review. You can see the screenshot above as a reference for what it'll look like

**Review by tdykstra — APPROVED** [review]
Thanks!

**Comment by nschonni** [comment]
Any blockers on this landing?

**Comment by adegeo** [comment]
LGTM

**Pull request merged**